### PR TITLE
Hip chat notifaction with url

### PR DESCRIPTION
**Pull request by kellyrob99**
Suggested format to include a jenkins url in notifications

### CODE REVIEW

File: jenkins/Jenkinsfile-build [hunk 1]
```diff
@@ -12,8 +12,7 @@ mavenSnapshotPipeline(
         onFailure: {
             build, env ->
                 hipchatSend color: 'RED', notify: false, room: 'nexus',
-                        message: "FAILURE: ${build.fullDisplayName}, " +
-                                "on branch ${env.BRANCH_NAME}, " +
-                                "duration: ${(build.duration/60000l) as int} min"
+                        message: """FAILURE: <a href="${env.BUILD_URL})">${build.fullDisplayName}</a>, on branch 
+${env.BRANCH_NAME}, duration: ${(build.duration / 60000l) as int } min"""
```
Comment (wwannemacher) [inline jenkins/Jenkinsfile-build]: should we have this consistent between the success and the fail notifications?

Comment (kellyrob99) [reply jenkins/Jenkinsfile-build]: can't hurt

File: jenkins/Jenkinsfile-build [hunk 2]
```diff
@@ -12,8 +12,7 @@ mavenSnapshotPipeline(
         onFailure: {
             build, env ->
                 hipchatSend color: 'RED', notify: false, room: 'nexus',
-                        message: "FAILURE: ${build.fullDisplayName}, " +
-                                "on branch ${env.BRANCH_NAME}, " +
-                                "duration: ${(build.duration/60000l) as int} min"
+                        message: """FAILURE: <a href="${env.BUILD_URL})">${build.fullDisplayName}</a>, on branch
```
Comment (mcculls) [inline jenkins/Jenkinsfile-build]: ^ appears to be a dangling `)` at the end of the URL?

File: jenkins/Jenkinsfile-build (hunk 1 above)
Comment (kellyrob99) [reply jenkins/Jenkinsfile-build]: @wwannemacher Sounds like this is going to be incorporated into https://github.com/sonatype/jenkins-shared/pull/30 so I'll close if there are no objections?

Comment (wwannemacher) [reply jenkins/Jenkinsfile-build]: yeah, I'm waiting for the exact same PR to incorporate for the NEXUS-16019-port-build-to-jenkins branch - I like the idea of having at least one layer of indirection knowing that hipchat's days are numbered.

**Pull request closed without merging**